MapleServer2/MapleServer2.csproj

```diff
@@ -18,6 +23,10 @@
       <PackageReference Include="NLog.Extensions.Logging" Version="1.6.5" />
       <PackageReference Include="Pastel" Version="2.1.0" />
       <PackageReference Include="SharpZipLib" Version="1.3.1" />
+      <PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.3">
+        <PrivateAssets>all</PrivateAssets>
+        <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      </PackageReference>
```

Comment:
And I didn't saw this package being used, it's needed?

Reply:
I tried to be able to step into functions provided by a dll, namely the mapletriggers. The debugger cannot step in those. I could not make it work yet.

Reply:
If you want you can import manually Maple2.Triggers instead of using the nuget package 👍